### PR TITLE
DS-303-avalanche-go-v-1-9-6-documentation-update

### PR DESCRIPTION
**Pull request by rob-ava**
### Changes

Change the AvalancheGo Config and Flags page to reflect the new name:
--whitelisted-subnets -> --track-subnets

See https://ava-labs.atlassian.net/browse/CPT-1290

### CODE REVIEW

**Comment by rob-ava** [comment]
Change available for preview here: https://rob-ava-solid-happiness-9v94g7q6679fx4jv-3000.preview.app.github.dev/nodes/maintain/avalanchego-config-flags

File: docs/nodes/maintain/avalanchego-config-flags.md [hunk 1]
```diff
@@ -601,11 +601,11 @@ Weight to provide to each peer when staking is disabled. Defaults to `1`.
 
 ## Subnets
 
-### Whitelist
+### Subnet Tracking
 
-#### `--whitelisted-subnets` (string)
+#### `--track-subnets` (string)
```
Comment (yulin-dong) [inline docs/nodes/maintain/avalanchego-config-flags.md:606]: -whitelisted-subnets still works, we should mark it deprecated, however,

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/nodes/maintain/avalanchego-config-flags.md (hunk 1 above)
Comment (rob-ava) [reply docs/nodes/maintain/avalanchego-config-flags.md:606]: I put it back but we should have a better strategy for deprecation.

File: docs/nodes/maintain/avalanchego-config-flags.md [hunk 2]
```diff
@@ -602,10 +602,14 @@ Weight to provide to each peer when staking is disabled. Defaults to `1`.
 ## Subnets
 
 ### Whitelist
-
 #### `--whitelisted-subnets` (string)
+Deprecated
```
Comment (yulin-dong) [inline docs/nodes/maintain/avalanchego-config-flags.md:606]: See example how to mark deprecation: https://docs.avax.network/apis/avalanchego/apis/c-chain#avaximportavax

**Review by yulin-dong — DISMISSED** [review]

**Review by yulin-dong — APPROVED** [review]

**Pull request merged**